### ISSUE TITLE
repo-updater: HTML version of repo-updater-state page

### ISSUE THREAD

**Issue by keegancsmith** [sourcegraph/sourcegraph#8656]
`repo-updater` currently exposes a useful debug endpoint for understanding what the git fetch scheduler is doing. However, the endpoint just dumps JSON. It would be great if instead we rendered a simple `html/template`:

- explain what each section is
- show human readable values of durations
- use html table

Example on our dogfood is at https://sourcegraph.sgdev.org/-/debug/repo-updater/repo-updater-state

Code is at https://github.com/sourcegraph/sourcegraph/blob/0c7c3e20efa5a65446fc1bb6a710c47fddf6ff10/cmd/repo-updater/shared/main.go#L217-L225

**Comment by ayzu** [comment]
Hi, I would like to take the issue. 

I was not able to sing in (using GitHub account) and see example at https://sourcegraph.sgdev.org/-/debug/repo-updater/repo-updater-state. I receive the following message:  

> Ask a site admin to create your account and then verify your email.

sourcegraph.sgdev.org has the following permissions in my GitHub account:

>  Full control of private repositories
>  Access user email addresses (read-only)

How can I get access to the example?

**Comment by unknwon** [comment]
Hi @ayzu, you have to have a `@sourcegraph` email address to be able to sign in our dogfood instance. However, you could see the example by running Sourcegraph instance locally (following https://docs.sourcegraph.com/#quickstart).

Once the server is up, visit http://localhost:7080/-/debug/repo-updater/repo-updater-state and you'll see:

![image](https://user-images.githubusercontent.com/2946214/75647026-93025c80-5c86-11ea-9f2f-892bea5b50df.png)

**Comment by anukul** [comment]
Hi! I would like to try this.

I've prepared a rough draft of the design:

![Safari -  repo-updater-state](https://user-images.githubusercontent.com/44864521/88492674-12901f80-cfca-11ea-9c12-f954aee90659.gif)

1. I think adding a new column for [`Repo.URL`](https://github.com/sourcegraph/sourcegraph/blob/3.18/cmd/repo-updater/repos/scheduler.go#L106) might look a bit weird in this design, so I've put a hyperlink on `Repo.Name`.  
Since the name might appear to be a link by itself, there's a tooltip to avoid confusion.

2. I added a tooltip on the Update Interval header to provide some explanation.  
Not exactly sure where else to add this - e.g. Priority, Sequence etc.
